airbyte-ci/connectors/live-tests/src/live_tests/regression_tests/conftest.py

```diff
@@ -121,6 +128,12 @@ def get_option_or_fail(request: SubRequest, option: str) -> str:
     pytest.fail(f"Missing required option: {option}")
 
 
+def ask_for_confirmation(message: str) -> None:
```

Comment:
Nice addition! Can you supply an option to override all the prompts from the test call locally? E.g. as a developer working on the regression test suite I probably want to `N` a lot during development each time I run the tests and then `Y` for good measure at the end, but it would be nice not to have to be interacting with the tests as they run each time

Reply:
I added a `deny-confirmation` option 👍